packages/bot-web-ui/src/pages/dashboard/announcements/announcements.scss

```diff
@@ -0,0 +1,197 @@
+@import '../../../styles/mixins.scss';
+
+.announcements {
+    position: absolute;
+    inset-inline-end: 2.4rem;
+    top: 1.8rem;
+    z-index: 1;
+
+    @include mobile-or-tablet-screen {
+        inset-inline-end: 1.6rem;
+    }
+
+    &__button {
+        @include flex-center;
+        padding: 0.8rem 1.4rem;
+        border: none;
+        outline: none;
+        border-radius: 2.4rem;
+        background-color: var(--general-section-1);
+        cursor: pointer;
+
+        @include mobile-or-tablet-screen {
+            height: 3.2rem;
+            width: 3.2rem;
+            line-height: 3.6rem;
+            text-align: center;
+            padding: 0;
+        }
+    }
+
+    &__icon {
+        height: 2.4rem;
+        width: 2.4rem;
+
+        @include mobile-or-tablet-screen {
+            height: 2.6rem;
+            width: 2.6rem;
+        }
+    }
+
+    &__label {
+        margin-inline-start: 0.4rem;
+    }
+
+    &__amount {
+        background-color: var(--status-danger);
+        border-radius: 0.9rem;
+        height: 1.8rem;
+        width: 1.8rem;
+        margin-inline-start: 0.4rem;
+        color: var(--brand-white);
+        font-weight: bold;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        @include mobile-or-tablet-screen {
+            position: absolute;
+            bottom: 2.1rem;
+            left: 1.8rem;
+        }
+    }
+}
+
+.notifications__wrapper {
+    padding-inline-start: 0.5rem;
+    border-inline-start: 1px solid var(--general-section-1);
+    border-width: 1px;
+    margin-inline-end: 1rem;
+
+    &--desktop {
+        z-index: 10;
+        top: 5rem;
+        right: 20rem;
```

Comment:
@maryia-matskevich-deriv I think this will break in RTL. Can you please check?

Reply:
@shafin-deriv Good point, thanks, code is added for this case✅